pom.xml

```diff
@@ -1499,7 +1499,21 @@
             <dependency>
                 <groupId>org.apache.kafka</groupId>
                 <artifactId>kafka-clients</artifactId>
-                <version>2.4.1</version>
+                <version>3.2.0</version>
```

Comment:
I would recommend avoiding to use "latest" in commit and PR title. It's not helpful when we see the history later.
Explicit number likes "Update kafka-clients version to 3.2.0" is better.

Reply:
PR title changed.
I'll make sure future PR's commit messages are as per your request

Reply:
Please also change the commit message (force pushing is ok).

I'll check the client release notes and get back with more comments if I have any.